src/app/component/ImageUpload/ImageUpload.test.js

```diff
@@ -1,11 +1,8 @@
 import React from "react";
-import { render, cleanup, fireEvent } from "@testing-library/react";
-import "jest-dom/extend-expect";
+import { render, fireEvent } from "@testing-library/react";
 import { ImageUpload } from "./ImageUpload";
 
 describe("ImageUpload Component", () => {
-  afterEach(cleanup);
```

Comment:
This is automatically setup with Jest, see: https://testing-library.com/docs/react-testing-library/api#cleanup